src/common/shared_config.h

```diff
@@ -11,8 +11,14 @@
 #define FW_UPDATE_ENABLE  0xAA
 #define FW_UPDATE_DISABLE 0x00
 
-// pin PA13 state
-#define APPENDIX_FLAG_MASK 0x01
+// appendix state
+typedef enum {
+    APNDX_EXISTS = 0, /**< Appendix not broken, signature authentication needed */
+    APNDX_BROKEN      /**< Appendix broken, no signature authentication needed */
+} BT_APNDX_STATUS;
```

Comment:
/suggestion/ Can we either have all the cases + the enums name prefixed with `BT_`, or none of them?

Also, I think it would be better if the name of this enum would correspond to the name of the „appendix location macro“.